modules/openfast-library/src/FAST_Lin.f90

```diff
@@ -5346,6 +5351,12 @@ SUBROUTINE FAST_CalcSteady( n_t_global, t_global, p_FAST, y_FAST, m_FAST, ED, BD
                m_FAST%Lin%AzimIndx = 1
                m_FAST%Lin%CopyOP_CtrlCode = MESH_UPDATECOPY
             end if
+            ! Forcing linearization if time is close to tmax
+            if ((t_global> p_FAST%TMax - 1.5*TwoPi_D/ED%y%RotSpeed) .and. .not.m_FAST%Lin%FoundSteady)  then
```

Comment:
Where does the 1.5 come from?  And I'm concerned that if the rotor speed is varying a lot during the trim solution, this condition may be met before the tail end of a simulation.  It would probably be better to use the initial rotor speed, which is the target rotor speed.

Reply:
I now replaced it with the Init rotor speed (this was a quick hack at first). 

I do need a margin, and it seems that the margin is actually 2 revolutions. The problem is that the condition is triggered right after the first target azimuth, it then has to wait until the next target azimuth is reached (which takes one revolution), and then perform the saving of operating points (over one revolution). This is not ideal, but should be fine before we have a steady-state solve.